src/utils/constants.js

```diff
@@ -22,10 +22,32 @@ export const helplineLinks = {
   english: "https://drive.google.com/open?id=1Mv0CuqmqV5jBv2sX2utuqXfPHIZ01_V_",
 };
 
+export const travelLinks = [
+  { link: "https://mapit.gov.in/covid-19", label: "Madhya Pradesh" },
+  { link: "https://www.digitalgujarat.gov.in/loginapp/CitizenLogin.aspx", label: "Gujarat" },
+  { link: "https://covidhelp.punjab.gov.in", label: "Punjab" },
+  { link: "https://covid19.mhpolice.in", label: "Maharashtra" },
+  { link: "https://emitraapp.rajasthan.gov.in/emitraApps/covid19MigrantRegistrationService", label: "Rajasthan" },
+  { link: "http://covidepass.hp.gov.in/", label: "Himachal Pradesh" },
+  { link: "http://tnepass.tnega.gov.in", label: "Tamil Nadu" },
+  { link: "https://edisha.gov.in/eForms/MigrantService", label: "Haryana" },
+  { link: "https://sevasindhu.karnataka.gov.in/Sevasindhu/English", label: "Karnataka" },
+  { link: "http://dsclservices.org.in/uttarakhand-migrant-registration.php", label: "Uttarakhand" },
+  { link: "https://covid19regd.odisha.gov.in/", label: "Odisha" },
+  { link: "goaonline.gov.in", label: "Goa " },
```

Comment:
https missing here

Reply:
The links i had received didn't contain https or http